src/wizards/tokenSwapDealWizard/stages/tokenDetailsStage/tokenDetailsStage.html

```diff
@@ -19,7 +19,6 @@ <h2 class="sidebarHeader heading heading3 title" data-test="section-title">
         <token-details
           hide-delete-button.bind="primaryDaoTokens.length === 1 && !isOpenProposalWizard"
           on-delete.call="deleteToken($index, primaryDaoTokens, primaryDAOTokenDetails, stageSettings.primaryDAOTokenDetailsViewModes)"
-          on-saved.call="checkedForUnsavedChanges()"
```

Comment:
Why remove this call altogether?

Reply:
because this check is already done by the validator